web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -292,38 +427,72 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
             label="Name"
             placeholder="Period Name"
             required
-            value={periodToAdd.name ? periodToAdd.name : ""}
-            onChange={(e) =>
-              setPeriodToAdd({ ...periodToAdd, name: e.target.value })
-            }
-          />
-          <TextField
-            className="fullWidth"
-            id="reviewPeriod-template-input"
-            label="Template ID"
-            placeholder="Template UUID"
-            required
-            value={periodToAdd.reviewTemplateId ? periodToAdd.reviewTemplateId : ""}
-            onChange={(e) =>
-              setPeriodToAdd({ ...periodToAdd, reviewTemplateId: e.target.value })
-            }
-          />
-          <TextField
-            className="fullWidth"
-            id="reviewPeriod-self-template-input"
-            label="Self Review Template ID"
-            placeholder="Template UUID"
-            required
-            value={periodToAdd.selfReviewTemplateId ? periodToAdd.selfReviewTemplateId : ""}
-            onChange={(e) =>
-              setPeriodToAdd({ ...periodToAdd, selfReviewTemplateId: e.target.value })
+            value={periodToAdd.name ? periodToAdd.name : ''}
+            onChange={e =>
+              setPeriodToAdd({...periodToAdd, name: e.target.value})
             }
           />
+          <div style={{margin: '1rem 0'}}>
+            <FormControl fullWidth>
+              <InputLabel id="template-label" required>
+                Review Template
+              </InputLabel>
+              <Select
+                label="Review Template"
+                labelId="template-label"
+                onChange={handleReviewTemplateChange}
+                required
+                value={
+                  periodToAdd.reviewTemplateId
+                    ? periodToAdd.reviewTemplateId
+                    : ''
+                }
+              >
+                <MenuItem key="empty" value="">
+                  -- Please Select --
```

Comment:
Is this standard text we want to display in all drop-downs when no item has been selected yet?

Reply:
Works for me!

Reply:
Fine by me as well.